src/screens/FeedScreen/ContentLink.js

```diff
@@ -57,7 +58,7 @@ const ContentLink = ({
   };
 
   return (
-    <View style={styles.contentFeed}>
+    <View style={styles.contentFeed(hasComment)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the `contentFeed` view now depends on the `hasComment` prop. This is a good use of conditional styling based on the component's props.



<!-- This is an auto-generated comment by OSS CodeRabbit -->